integreat_cms/cms/views/utils/machine_translations.py

```diff
@@ -0,0 +1,83 @@
+"""
+This module contains the generalized function to build the AJAX call for the machine translation popup
+"""
+import json
+
+from ....textlab_api.utils import check_hix_score
+from ...models import Page
```

Comment:
```suggestion
import json

from django.http import Http404, HttpResponseNotFound, JsonResponse

from ....textlab_api.utils import check_hix_score
from ...models import Event, Page, POI
```

Reply:
Add to __init__ file

from .machine_translations import build_json_for_machine_translation